lib/util/fs/index.spec.ts

```diff
@@ -37,17 +36,30 @@ jest.mock('find-up');
 const findUp = mockedFunction(_findUp);
 
 describe('util/fs/index', () => {
-  let dirResult: DirectoryResult;
+  let localDirResult: DirectoryResult;
+  let localDir: string;
+
+  let cacheDirResult: DirectoryResult;
+  let cacheDir: string;
+
+  let tmpDirResult: DirectoryResult;
   let tmpDir: string;
 
   beforeEach(async () => {
-    GlobalConfig.set({ localDir: '' });
-    dirResult = await tmp.dir({ unsafeCleanup: true });
-    tmpDir = dirResult.path;
+    localDirResult = await tmp.dir({ unsafeCleanup: true });
+    localDir = localDirResult.path;
+
+    cacheDirResult = await tmp.dir({ unsafeCleanup: true });
+    cacheDir = cacheDirResult.path;
+
+    tmpDirResult = await tmp.dir({ unsafeCleanup: true });
+    tmpDir = tmpDirResult.path;
+
+    GlobalConfig.set({ localDir, cacheDir });
   });
 
   afterEach(async () => {
-    await dirResult.cleanup();
+    await localDirResult.cleanup();
```

Comment:
```suggestion
    await localDirResult?.cleanup();
    await cacheDirResult?.cleanup();
    await tmpDirResult?.cleanup();
```

Don't cause error if tmp couldn't created and cleanup temp dirs

Reply:
Good catch